src/Azure/Orleans.Persistence.AzureStorage/Providers/Storage/AzureTableStorage.cs

```diff
@@ -202,23 +203,29 @@ private static async Task DoOptimisticUpdate(Func<Task> updateOperation, string
         /// </remarks>
         internal void ConvertToStorageFormat<T>(T grainState, TableEntity entity)
```

Comment:
It would be great to add a test to check the behavior of the new option

Reply:
Yep